packages/backend/src/instanceMetadata.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  coreServices,
+  createBackendPlugin,
+} from '@backstage/backend-plugin-api';
+import { EXPERIMENTAL_instanceMetadataServiceRef } from '@backstage/backend-plugin-api/alpha';
+
+// Example usage of the instance metadata service to log the installed features.
+export default createBackendPlugin({
```

Comment:
Happy to have this one installed by default too 😁

Reply:
done in [c1a3bdb](https://github.com/backstage/backstage/pull/27921/commits/c1a3bdbe1276f7a0a88861f41eff00662b2d9bc5) 😁